### ISSUE TITLE
After unlock the achievement doesn't count in my profile

### ISSUE THREAD

**Issue by 7amoksha26** [gibbed/SteamAchievementManager#391]
Hello guys,
when i use SAM for specific achievement it appears in library i do it, but in profile still not unlocked.
what is the solve for this problem?

**Comment by ach14624** [comment]
Yes, this has been happening more often with the latest Steam updates, usually when you open a game with SAM and unlock your first achievement it does not sync online at all, the achievement is only unlocked locally, almost as if Steam is not connected online.

The way I have found to trigger it to sync online properly is to lock and unlock it twice, but this is obviously far from ideal so I would love to hear if there is a better fix for this problem.
